errors.go

```diff
@@ -339,12 +340,12 @@ func (e *RFC6749Error) Sanitize() *RFC6749Error {
 func (e *RFC6749Error) GetDescription() string {
 	description := e.Description
 	if e.Hint != "" {
-		description += "\n\n" + e.Hint
+		description += " " + e.Hint
 	}
 	if e.Debug != "" {
-		description += "\n\n" + e.Debug
+		description += " " + e.Debug
 	}
-	return description
+	return strings.ReplaceAll(description, "\"", "'")
```

Comment:
Also `\` is not allowed, and anything non-ASCII, so I think we have to sanitize this more.

Reply:
Right, my idea here was to just remove this for any stray characters but I guess we could santize it even more - but would probably mark this as a feature instead of addressing this here.

Reply:
https://github.com/ory/fosite/issues/527